Remove unecessary space in code

johanf219 opened the pull request:
Remove space on code

Commit:
Remove unecessary space in code

JOHAN

Commit:
Add connection file

What: Add connection file on database folder
Why: Its necessary to connect with the database

JOHAN

Commit:
Add function to store users

Why: Its necessary to store users on Users table

JOHAN

Commit:
Add user file

What: This file will contain the functions to store, load and update the users
Why: Functionality and readability

JOHAN

Commit:
Add import in user file

JOHAN

Commit:
Remove changes on .idea folder

What: Remove bad declaration on .iml
Why: When i installed javafx some files changed i dont know why

JOHAN

Commit:
Rename file to add .java

JOHAN

Commit:
Add functinality to saveUserOnTable

What: Store information on users table
Why: Its necessary to functionality

JOHAN

Commit:
Add function to load the user information

What: Load information from users table
Why: It will be necessary in the future

JOHAN

diego7785 commented on database/connection.java:
```diff
@@ -0,0 +1,15 @@
+        try {
+            String host = "jdbc:mysql://localhost:3306/Desarrollo";
+            String userName = "root";
+            String userPassword = "Johan219";
```
Esto debería ser general, no para un desarrollador específico

diego7785 commented on database/connection.java:
```diff
@@ -0,0 +1,15 @@
+public class DBConnect {
+    public static void main(String[] args) {
+        try {
+            String host = "jdbc:mysql://localhost:3306/Desarrollo";
```
Igual aquí, el localhost debería ser parámetro

Commit:
Modify the constructor

What: The constructor will receive the host, user name and password to connect with the database
Why: Its necessary to functionality

JOHAN
